docs/nginx.md

```diff
@@ -57,13 +57,19 @@ Read again [the requirements](recipes.md#requirements).
 
 Ready?
 
-Run the following:
+--
+
+Create the required directories
 
 ```
 mkdir -p auth
 mkdir -p data
+```
+
+Create the main nginx configuration you will use.
+
+```
 
-# This is the main nginx configuration you will use
 cat <<EOF > auth/nginx.conf
```

Comment:
We could but I like that you can simply copy and paste the block. I just
felt that having all the steps in one huge block was unreadable.
On Dec 7, 2015 6:23 PM, "Stephen Day" <notifications@github.com> wrote:

> In docs/nginx.md
> <https://github.com/docker/distribution/pull/1244#discussion_r46904114>:
>
> >
> > -# This is the main nginx configuration you will use
> >  cat <<EOF > auth/nginx.conf
>
> If we are splitting this up, do we want to remove the whole cat flow?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/distribution/pull/1244/files#r46904114>.
>